src/zzz_od/application/driver_disc_read/driver_disc_read_app.py

```
@@ -0,0 +1,627 @@
+                else:
+                    log.warning(f'驱动盘识别失败 [{global_index}]')
+
+            except Empty:
+                continue
+            except Exception as e:
+                log.error(f'结果收集异常: {e}', exc_info=True)
+
+        log.info('结果收集线程已退出')
+
+    def _get_panel_signature(self) -> float:
+        """获取当前驱动盘详情面板的签名（用于检测面板是否更新）"""
```

Ruff (0.14.0)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)